src/components/plots/MultiViewGrid.jsx

```diff
@@ -1,39 +1,137 @@
   const dispatch = useDispatch();
 
   const cellSets = useSelector(getCellSets());
 
-  const multiViewConfig = useSelector((state) => state.componentConfig[multiViewUuid].config);
-  const plotConfigs = useSelector(getPlotConfigs(multiViewConfig.plotUuids));
+  const multiViewConfig = useSelector((state) => state.componentConfig[multiViewUuid]?.config);
+  const plotConfigs = useSelector(getPlotConfigs(multiViewConfig?.plotUuids));
+  const multiViewPlotUuids = multiViewConfig?.plotUuids;
+  const firstPlotUuid = `${plotUuid}-0`;
 
   const [plots, setPlots] = useState({});
   const previousMultiViewConfig = useRef({});
+  const shownGenes = _.compact(multiViewPlotUuids?.map((uuid) => plotConfigs[uuid]?.shownGene));
+  const highestDispersionGene = useSelector(
+    (state) => state.genes.properties.views[plotUuid]?.data[0],
+  );
+
+  const debounceSaveAll = useCallback(_.debounce(() => {
+    const allComponentUuids = _.concat(multiViewUuid, multiViewPlotUuids);
+    allComponentUuids.forEach((uuid) => {
+      if (uuid) {
+        dispatch(savePlotConfig(experimentId, uuid));
+      }
+    });
+  }, 2000), [multiViewConfig]);
+  const expression = useSelector((state) => state.genes.expression.full);
+
+  const loadComponent = (componentUuid, type, skipAPI, customConfig) => {
+    dispatch(loadConditionalComponentConfig(
+      experimentId, componentUuid, type, skipAPI, customConfig,
+    ));
+  };
+
+  useEffect(() => {
+    if (!highestDispersionGene) {
+      dispatch(loadPaginatedGeneProperties(experimentId, PROPERTIES, plotUuid, tableState));
+    }
+    if (!multiViewConfig) {
+      loadComponent(multiViewUuid, multiViewType, false);
+    }
+  }, []);
+
+  useEffect(() => {
+    if (!multiViewConfig) return;
+
+    debounceSaveAll();
+  }, [plotConfigs, multiViewConfig]);
+
+  useEffect(() => {
+    if (plotConfigs[firstPlotUuid]
+      && !plotConfigs[firstPlotUuid]?.shownGene
+      && highestDispersionGene) {
+      dispatch(updatePlotConfig(firstPlotUuid,
+        { shownGene: highestDispersionGene, title: { text: highestDispersionGene } }));
+    }
+  }, [plotConfigs]);
+
+  useEffect(() => {
+    // initial set up if there are no plots
+    // plotting one plot using the highest dispersion gene
+
+    if ((multiViewConfig && highestDispersionGene && !multiViewPlotUuids?.length)) {
+      dispatch(loadGeneExpression(experimentId, [highestDispersionGene], firstPlotUuid));
+
+      const customMultiPlotConfig = { plotUuids: [firstPlotUuid] };
+      loadComponent(multiViewUuid, multiViewType, true, customMultiPlotConfig);
+
+      const customFirstPlotConfig = { shownGene: highestDispersionGene, title: { text: highestDispersionGene } };
+      loadComponent(firstPlotUuid, plotType, true, customFirstPlotConfig);
+    }
+  }, [multiViewConfig, highestDispersionGene, plotConfigs]);
+
+  useEffect(() => {
+    if (!shownGenes?.length) return;
+
+    const unLoadedGenes = shownGenes.filter((gene) => (
```

Comment:
nit-picky comment: unloaded is one word, so it makes sense for the variable to be called `unloadedGenes`.